internal/services/serviceconnector/sdk/2022-05-01/links/client.go

```diff
@@ -0,0 +1,15 @@
+package links
```

Comment:
@jiaweitao001 where's this SDK come from? is there an upstream PR to add support for it?

Reply:
Hi @tombuildsstuff, this SDK was generated by pandora sdk tool.  There's no upstream PR.